src/applications/drydock/blueprint/DrydockWorkingCopyBlueprintImplementation.php

```diff
@@ -7,7 +7,7 @@ final class DrydockWorkingCopyBlueprintImplementation
   const PHASE_REMOTEFETCH = 'blueprint.workingcopy.fetch.remote';
   const PHASE_MERGEFETCH = 'blueprint.workingcopy.fetch.staging';
   // TM CHANGES BEGIN
-  const PHASE_MERGEREBASE = 'blueprint.workingcopy.rebase';
+  const TMP_BRANCH_PREFIX = 'tmp-drydock';
```

Comment:
Is this used anywhere?

Reply:
oops good spot I renamed it from GIT_BRANCH_PREFIX... I'll change it everywhere else